packages/mobile/src/screens/audio-screen/Panel.tsx

```diff
@@ -72,7 +75,8 @@ export const Panel = ({
   progressLabel,
   remainingLabel,
   buttonInfo,
```

Comment:
can we delete this arg if it's no longer used?

Reply:
good eye